src/platform/src/test/java/org/geoserver/platform/resource/PathsTest.java

```diff
@@ -148,23 +162,35 @@ public void parentTest() {
     }
 
     @Test
-    public void naming() {
+    public void naming() throws IOException {
         assertEquals("file.txt", name("directory/file.txt"));
         assertEquals("txt", extension("directory/file.txt"));
 
         assertEquals("directory/file.txt", sidecar("directory/file", "txt"));
         assertEquals("directory/file.prj", sidecar("directory/file.txt", "prj"));
+
+        File home = new File(System.getProperty("user.home")).getCanonicalFile();
+        String absolutePath = Paths.convert(home.getPath());
+
+        assertTrue("home", Paths.isAbsolute(absolutePath));
+        String root = Paths.names(absolutePath).get(0);
+        assertTrue("system root '" + root + "'", Paths.isAbsolute(root));
     }
 
     @Test
-    public void convert1() {
+    public void convert1() throws IOException {
+        // relative paths
         File folder = new File("folder");
         File file1 = new File("file1");
         File file2 = new File(folder, "file2");
 
         assertEquals("folder", Paths.convert(folder.getPath()));
         assertEquals("folder/file2", Paths.convert(file2.getPath()));
         assertEquals("file1", Paths.convert(file1.getPath()));
+
+        // absolute paths
+        File home = new File(System.getProperty("user.home")).getCanonicalFile();
+        assertTrue(isAbsolute(Paths.convert(home.getPath())));
```

Comment:
@aaime this is the test that covers use of windows absolute path (to ensure ability to handle windows drive letter correctly).